packages/cozy-harvest-lib/README.md

```diff
@@ -77,3 +99,9 @@ Any konnector can provide new locales for this field labels or add new ones. The
 ## OAuthForm
 
 The OAuthForm is used for OAuth konnector and display a single "Connect" button.
+
+
+## Doc and example about manifests
+
+[https://docs.cozy.io/en/tutorials/konnector/#the-manifest]
```

Comment:
Why put that into brackets here?

Reply:
( confusion with some wiki markdown style ... ^^U )